h2integrate/simulation/technologies/hydrogen/electrolysis/run_h2_PEM.py

```diff
@@ -198,6 +201,45 @@ def run_h2_PEM(
     if not debug_mode:
         h2_ts, h2_tot = clean_up_final_outputs(h2_tot, h2_ts)
 
+    n_stacks_new = int(sum(np.isnan(h2_tot.loc["Stack Life [hours]"].to_list())))
+
+    # if some stacks were never turned on (are "new") there may be inf and nans in some outputs
+    if n_stacks_new > 0:
+        never_on_cluster_list = np.isnan(h2_tot.loc["Stack Life [hours]"].to_list())
+        user_defined_pem_param_dictionary.setdefault("curve_coeff", None)
+        pem = create_1MW_reference_PEM(curve_coeff=user_defined_pem_param_dictionary["curve_coeff"])
```

Comment:
Why do you create this `pem` object here? Is it just to access the `pem.eta_h2_hhv` attribute later? If so, I'd suggest getting that value a different way. We could:
- save `eta_h2_hhv` in the results dict and access it that way
- because `eta_h2_hhv` is hardcoded, just define it once in `PEM_H2_LT_electrolyzer_Clusters.py` outside of the `PEM_H2_Clusters` class and import it from that file for use here

This way we avoid creating an object unnecessarily.